### DISCUSSION TITLE
How to exclude files?

### DISCUSSION THREAD

**Discussion by markcellus** [jprichardson/reload#321] [Q&A] [answered]
Is there a way to exclude files? We have a project that has files in the root directory (and subdirectories) that we don't want reloading the page when edited. I see the ideas in https://github.com/alallier/reload/issues/25#issuecomment-243209492 but they seems to only apply when supervisor is being used. 

@jprichardson @alallier any thoughts? 

Thanks!

**Comment by alallier** [comment, marked as answer]
As I touched on [here](https://github.com/alallier/reload/discussions/320#discussioncomment-4576649) when using Reload in an Express app it's only concern is reload the client side of things. Whatever you have "watching" your server files and reloading the server will need to do the ignore.
